packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+trying it out, developing and managing services, and applying product settings.
+Within **Monitor**, a user should be able to find all content related to monitoring their application,
+such as logs and insights.
+
+On a rare occasion, there are tasks that span between Develop and Monitor.
+For example, a user may learn that calls are not connecting in India through Voice Insights (in **Monitor**),
+and that they need to turn on India in their Voice Geo-permissions (in **Develop**). In cases like these,
+good interlinking between the relevant pages or contextual quick actions will provide a streamlined
+user experience. In addition, the tabs will hold the menu states so that less clicks are required to drill
+in and between menus under each tab.
+
+![](../../../assets/images/experiences/console/develop-or-monitor.png)
+
+<ImageCaption>
+  A diagram of the Console experience, showing the steps from left to right as Evaluate, Develop, Monitor, and Grow. An
+  arrow points from Explore, a subsection below Grow, to Evaluate (Existing user), a subsection below Develop, an
+  example of good interlinking.
+</ImageCaption>
+
+---
+
+### What goes in Develop
+
+In Console’s "Develop" tab, each product's navigation structure is organized into 4 categories:
+
+**1. Overview**
+
+This single page should provide information about what this product is, what customer problems it can help
+solve, how it works with other Twilio products, key features, available regions, link to pricing,
+and links to get started. Please utilize the [Overview Page Template](https://docs.google.com/document/d/1OA9JSj1KA8CqeFtArwcbyK2lKYV_4Oyx2CPOcqzGHRc/edit?usp=sharing)
+for this purpose.
+
+**2. Try it out (optional)**
+
+This page or section is optionally used in a product's navigation when an interactive setup flow or tutorial
+separate from twilio.com/docs is required in order to gain access to the product.
+(Example: Messaging > Try it out > Whatsapp). It can also be used if there is an interactive sandbox
+experience that allows you to interact with the APIs. This page should not be used to duplicate twilio.com/docs content.
+
+**3. Manage**
+
+This is the core of the product. All services that are required to run an application fall into this category.
+The "Manage" label is used when there are 3 or more pages related to the configuration and management of the
+product. (Example: Voice). If you have just 1-2 "Manage" pages, you do not need to use the ‘Manage' folder.
+(Example: Messaging which has just "Services" &amp; "Senders" pages).
+
+### What goes in Monitor
+
+The monitor tab is the destination for all pages related to (you guessed it!) monitoring your Twilio application.
+This includes the logs, and insights today. Logs and Insights pages appear under Monitor for the products
+you use or pin to your side navigation.
+
+#### Logs
+
+**Primary Personas:** Developers, Builders, Technical support managers
```

Comment:
For all of the sections that have the "Primary/secondary personas" + "Jobs to be done" lines, is it possible to have them in the same paragraph so they don't have the big chunky margin between them, like so?

![image](https://user-images.githubusercontent.com/5374045/166962057-90787d14-3cd3-48cb-a0c8-7eb3bc8c129f.png)